.azure-pipelines/weekly-generation.yml

```diff
@@ -1,170 +1,192 @@
   BuildAgent: ${{ parameters.BuildAgent }}
   SkipForceRefresh: ${{ parameters.SkipForceRefresh }}
-
-pool: $(BuildAgent)
 trigger: none
 pr: none
 schedules:
-  - cron: "0 12 * * TUE" # Run every Tuesday at noon UTC
-    displayName: "PS SDK Weekly Refresh"
-    branches:
-      include:
-        - dev
-    always: true
-jobs:
-  - job: RefreshOpenAPIDocuments
-    displayName: Refresh OpenApi documents
-    timeoutInMinutes: 240
-    pool: $(BuildAgent)
-    steps:
-      - template: common-templates/download-openapi-docs.yml
-        parameters:
-          Branch: $(Branch)
-          BaseBranch: $(BaseBranch)
-          BuildAgent: $(BuildAgent)
-          SkipForceRefresh: $(SkipForceRefresh)
-          SkipOpenAPIDocsDownload: ${{ parameters.SkipOpenAPIDocsDownload }}
-
-  - job: MsGraphPsSdkWeeklyGeneration
-    dependsOn: RefreshOpenAPIDocuments
-    displayName: Microsoft Graph PowerShell SDK Generation
-    condition: and(succeeded(), ne(dependencies.RefreshOpenAPIDocuments.outputs['OpenAPIDocDiff.ModulesWithChanges'], ''))
-    timeoutInMinutes: 840
-    variables:
-      WeeklyBranch: $[ dependencies.RefreshOpenAPIDocuments.outputs['ComputeBranch.WeeklyBranch'] ]
-    steps:
-      - template: ./common-templates/checkout.yml
-        parameters:
-          TargetBranch: $(WeeklyBranch)
-      - template: ./common-templates/install-tools.yml
-      - template: ./common-templates/security-pre-checks.yml
-      - template: ./generation-templates/authentication-module.yml
-        parameters:
-          Test: ${{ parameters.Test }}
-          Pack: ${{ parameters.Pack }}
-          Sign: ${{ parameters.Sign }}
-      - template: ./generation-templates/workload-modules.yml
-        parameters:
-          Test: ${{ parameters.Test }}
-          Pack: ${{ parameters.Pack }}
-          Sign: ${{ parameters.Sign }}
-      - template: ./generation-templates/meta-module.yml
-        parameters:
-          Test: ${{ parameters.Test }}
-          Pack: ${{ parameters.Pack }}
-          Sign: ${{ parameters.Sign }}
-      - ${{ if and(eq(parameters.Pack, true), eq(parameters.Sign, true)) }}:
-          - template: ./common-templates/esrp/codesign-nuget.yml
-            parameters:
-              FolderPath: "$(Build.ArtifactStagingDirectory)"
-              Pattern: "Microsoft.Graph*.nupkg"
-          - task: PublishBuildArtifacts@1
-            displayName: Publish Module Artifacts
-            inputs:
-              PathtoPublish: "$(Build.ArtifactStagingDirectory)"
-              ArtifactName: "drop"
+- cron: "0 12 * * TUE"
+  displayName: "PS SDK Weekly Refresh"
+  branches:
+    include:
+    - dev
+  always: true
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
```

Comment:
@timayabi2020  I think there is a big difference in agent capabilities which may be the cause of the timeouts. The previous pool is using `1es-windows-ps-compute` while this is changing to `Azure-Pipelines-1ESPT-ExDShared` which has only 8GB and 2 CPUs

Reply:
Oh nice. Where did you get that info from?

Reply:
Sorry. I thought I pasted this here. 
https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/migration/ed-sharedpool#azure-pipelines-1espt-exdshared